iron-overlay-behavior.html

```diff
@@ -15,7 +15,7 @@
 <link rel="import" href="iron-overlay-manager.html">
 
 <script>
-
+'use strict';
```

Comment:
I see. Well, shouldn't this be handled by the vulcanization script? I don't go crazy about adding an IFEE just for concatenation :/